modules/ROOT/partials/configuration/toolbar_mode.adoc

```diff
@@ -6,7 +6,7 @@ NOTE: The `toolbar_drawer` option was deprecated with the release of {productnam
 
```

Comment:
There's errors being thrown in the logs about this file that need fixing as well. I believe the cause is because `Settings` was made discrete below when it shouldn't have been.
```
[08:18:20.042] WARN (asciidoctor): section title out of sequence: expected level 2, got level 3
    file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/partials/configuration/toolbar_mode.adoc:50
    source: /Users/lee.newson/git/tinymce-docs (refname: HEAD <worktree>, start path: .)
    stack:
        file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/pages/configure/editor-appearance.adoc:84
[08:18:20.044] WARN (asciidoctor): section title out of sequence: expected level 2, got level 3
    file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/partials/configuration/toolbar_mode.adoc:70
    source: /Users/lee.newson/git/tinymce-docs (refname: HEAD <worktree>, start path: .)
    stack:
        file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/pages/configure/editor-appearance.adoc:84
[08:18:20.046] WARN (asciidoctor): section title out of sequence: expected level 2, got level 3
    file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/partials/configuration/toolbar_mode.adoc:90
    source: /Users/lee.newson/git/tinymce-docs (refname: HEAD <worktree>, start path: .)
    stack:
        file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/pages/configure/editor-appearance.adoc:84
[08:18:20.048] WARN (asciidoctor): section title out of sequence: expected level 2, got level 3
    file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/partials/configuration/toolbar_mode.adoc:114
    source: /Users/lee.newson/git/tinymce-docs (refname: HEAD <worktree>, start path: .)
    stack:
        file: /Users/lee.newson/git/tinymce-docs/modules/ROOT/pages/configure/editor-appearance.adoc:84
```